docs/architecture/index.md

```diff
@@ -0,0 +1,130 @@
+
+**Modular Architecture**: designed as a modular platform, with separate components for data collection, processing, and storage. This modular architecture enables developers to add new features and capabilities to the platform without disrupting the core functionality.
+
+**Asynchronous Processing**: Zentral uses asynchronous processing techniques, such as Celery, RabbitMQ and Google PubSub to handle incoming data and alerts. This approach ensures that the platform can handle large volumes of data and respond quickly to security events.
+
+**Storage Flexibility**: the platform supports a range of different technologies to store events, including Elasticsearch, Kinesis, Splunk, Humio, OpenSearch and Azure. This flexibility enables organizations to choose the database technology that best fits their needs and requirements.
+
+**RESTful API**: Zentral provides a RESTful API that enables developers to build custom integrations and extensions, as well as to automate tasks and workflows within the platform (e.g. reporting).
+
+**Docker-Based Deployment**: The platform is designed to be deployed using [Docker](/deployment/docker-compose), a containerization technology that provides a standardized and reproducible way of deploying and managing the platform.  [Images on AWS and GCP: ZAIO (Zentral all in one)](/deployment/) are also provided.
+
+## System performance and scalability
+
+Cloud Based deployments, using [ZAIO (Zentral all in one images)](/deployment/) in AWS and GCP. Also Terraform configuration is provided for Enterprise and SaaS plans.
+
+With Django Framework using PostgreSQL, the database can be scaled vertically and horizontally independently. Django also support RabbitMQ/Celery for Asynchronous processing to support high loads.
+
+## Security and privacy
+
+Zentral ensures the **confidentiality**, **integrity**, and **availability** of endpoint data. Here are some of the key security and privacy features implemented:
+
+**Secure connections**: all data transmitted between the endpoints and Zentral is encrypted using **TLS** and **mTLS**.
+
+**Encryption**: Zentral can be configured to [**encrypt some DB fields**](/configuration/secret_engines/) that are considered secrets. Secret engines supported: AWS Key Mgmt Service, Fernet backend, Google Cloud Key Mgmt.
+
+**MDM requires multiple certificates to operate**, including an **APNs certificate** to talk to clients (push notifications) and an **SSL certificate** to communicate securely.
+
+Django provides default security characteristics (API tokens, Authentication, password hashes). For authentication, [Okta](/configuration/okta_saml/) (or any other SAMLs) can be integrated with Zentral, hence Two-Factor-Authentication (**2FA**) is supported.
```

Comment:
Identity providers instead of SAMLs.

Two factor (WebAuthN, TOTP) also supported natively for the local accounts (not from the IdP).

Reply:
Applied your suggestion and added the last phrase.